.github/workflows/ci.yml

```
@@ -0,0 +1,46 @@
+      - name: Generate application-secret.yml # 비밀 설정 파일 생성
+        run: |
+          cat > src/main/resources/application-secret.yml <<EOL
+          openweather:
+            api:
+              key: ${{ secrets.OPENWEATHER_API_KEY }}
+          EOL
+
+      - name: Run tests # 테스트 실행
+        run: ./gradlew test --no-daemon
+        env:
+          SPRING_PROFILES_ACTIVE: test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)